src/components/inputs/dropdown.js

```diff
@@ -42,7 +42,7 @@ export default class Dropdown extends React.Component {
 
     render() {
 
-        let {onChange, value, className, error, clearable, disabled, overrideCSS, ...rest} = this.props;
+        let {onChange, value, className, error, clearable, disabled, overrideCSS, ariaLabelledby, ...rest} = this.props;
```

Comment:
what if the user does not provides the attr ? i do think we should think about that case